db/migrate/20150827124834_alter_subject_workflow_counts_add_subject_id.rb

```diff
@@ -0,0 +1,33 @@
+class AlterSubjectWorkflowCountsAddSubjectId < ActiveRecord::Migration
+  def up
+    add_column :subject_workflow_counts, :subject_id, :integer, index: true
+    add_foreign_key :subject_workflow_counts, :subjects
+
+    # Create new aggregated (per subject) SWC records
+    execute <<-END
+      INSERT INTO subject_workflow_counts (subject_id, workflow_id, classifications_count, created_at, updated_at, retired_at)
+      SELECT sms.subject_id,
+             workflow_id,
+             SUM(classifications_count) AS classifications_count,
```

Comment:
Do we want to run a sum here? I thought we updated each sms for a subject / workflow combo so in theory they should have the same classification count.